lib/modules/manager/sbt/extract.ts

```diff
@@ -32,10 +37,41 @@ interface Ctx {
   depType?: string;
   useScalaVersion?: boolean;
   variableName?: string;
+
+  sbtVersion?: string;
 }
 
 const scala = lang.createLang('scala');
 
+const sbtVersionMatch = q
+  .sym<Ctx>('sbt')
+  .op('.')
+  .sym('version')
+  .op('=')
+  .num((ctx, { value: sbtMajor }) => ({ ...ctx, sbtVersion: sbtMajor }))
+  .num((ctx, { value: sbtMinor }) => ({
+    ...ctx,
+    sbtVersion: (ctx.sbtVersion as string) + sbtMinor,
```

Comment:
```suggestion
    sbtVersion: ctx.sbtVersion! + sbtMinor,
```
...if you do it via regex instead, this will be obsolete of course